src/main/java/passableleaves/core/PLCore.java

```diff
@@ -27,7 +29,7 @@ public PLCore() {
 
     @Override
     public String[] getASMTransformerClass() {
-        return new String[0];
+        return new String[] {"passableleaves.core.PLTransformer"};
```

Comment:
From what i _remember_ from testing, Transformers get the obfuscated names passed into the .transform call instead of the SRG names.. idk if it affects anything here and i'm kinda too busy to double-check at the moment... Thought i'd mention it before anything breaks and noone knows why it suddenly happens.